plugins/request/src/main/java/org/apache/unomi/plugins/request/actions/SetRemoteHostInfoAction.java

```diff
@@ -243,7 +243,7 @@ private static boolean isAValidIPAddress(String remoteAddr) {
             try {
                 addr = InetAddress.getByName(remoteAddr);
             } catch (UnknownHostException e) {
-                logger.debug("Cannot resolve IP", e);
+                logger.warn("Cannot resolve IP", e);
```

Comment:
use debug to log the stack trace and only display  the error message as a warning